compiler/hie-core/src/Development/IDE/Core/FileStore.hs

```diff
@@ -129,8 +129,8 @@ getFileContentsRule vfs =
         res <- liftIO $ ideTryIOException file $ do
             mbVirtual <- getVirtualFile vfs $ filePathToUri' file
             case mbVirtual of
-                Just (VirtualFile _ rope _) -> return $ textToStringBuffer $ Rope.toText rope
-                Nothing -> hGetStringBuffer (fromNormalizedFilePath file)
+                Just (VirtualFile _ rope _) -> return $ Just $ textToStringBuffer $ Rope.toText rope
```

Comment:
This looks like it is just `pure $ fmap (\(VirtualFile _ rope _) -> textToStringBuffer $ Rope.toText rope) mbVirtual`

Reply:
Done